bin/http-server

```diff
@@ -45,14 +46,32 @@ if (argv.h || argv.help) {
   process.exit();
 }
 
-var port = argv.p || argv.port || parseInt(process.env.PORT, 10),
-    host = argv.a || '0.0.0.0',
-    ssl = !!argv.S || !!argv.ssl,
-    proxy = argv.P || argv.proxy,
-    utc = argv.U || argv.utc,
+var root = config.get('_')[0];
+
+if (!root) {
+  try {
+    fs.lstatSync('./public');
+    root = './public';
+  }
+  catch (err) {
+    root = './';
+  }
+}
+
+config.file({
+  file: '.httpserverrc',
+  dir: path.resolve(root),
+  search: true
+});
+
+var port = config.get('p') || config.get('port') || parseInt(process.env.PORT, 10),
+    host = config.get('a') || '0.0.0.0',
+    ssl = !!config.get('S') || !!config.get('ssl'),
```

Comment:
I _think_ we can use `parseValues: true` in `.argv` above rather than need to coerce here. Correct me if I'm wrong on that, I haven't used nconf before.